src/main/java/pl/wrapper/parking/infrastructure/error/HandleResult.java

```diff
@@ -35,6 +34,8 @@ private static ErrorWrapper getInfoByError(Error error, String uri, HttpStatus o
                     "Parking of name: " + e.name() + " not found", onSuccess, uri, HttpStatus.NOT_FOUND);
             case ParkingError.ParkingNotFoundByAddress e -> new ErrorWrapper(
                     "Parking of address: " + e.address() + " not found", onSuccess, uri, HttpStatus.NOT_FOUND);
+            case ParkingError.ParkingWithTheMostFreeSpotsNotFound ignored -> new ErrorWrapper(
```

Comment:
Let's change the error name and message. Since no results were found, no parking lots have any free places remaining. "ParkingWithTheMostFreeSpotsNotFound" -> something along the lines of "no free parking spots available"

Reply:
> Let's change the error name and message. Since no results were found, no parking lots have any free places remaining. "ParkingWithTheMostFreeSpotsNotFound" -> something along the lines of "no free parking spots available"

So for example change the name of ParkingError record to NoFreeParkingSpotsAvailable?
case ParkingError.NoFreeParkingSpotsAvailable ignored -> new ErrorWrapper(
                    "No free parking spots available", onSuccess, uri, HttpStatus.NOT_FOUND);
Do I understand it correctly?

Reply:
Yes. If there was no parking lot found with most free spots, then it means there are no free spots available at all